.changes/fix-airtable-filenames.md

```diff
@@ -0,0 +1,9 @@
+---
+"gatsby-source-airtable": patch
```

Comment:
```suggestion
"gatsby-source-airtable": minor
```

I think this change should be relatively safe, but I suspect there is enough of a change here that it might be worth moving this to a minor. We are considering both a more "true" file extension and the filename instead of whatever `gastby-source-filesystem` was giving us.